Disable build of googletest to avoid build-time conflict with
devel/gtest if installed.

Index: node.gyp
--- node.gyp.orig
+++ node.gyp
@@ -826,7 +826,7 @@
       ],
       'dependencies': [
         'deps/base64/base64.gyp:base64',
-        'deps/googletest/googletest.gyp:gtest_prod',
+        # 'deps/googletest/googletest.gyp:gtest_prod',
         'deps/histogram/histogram.gyp:histogram',
         'deps/uvwasi/uvwasi.gyp:uvwasi',
         'deps/simdutf/simdutf.gyp:simdutf',
@@ -1144,8 +1144,8 @@
       'dependencies': [
         '<(node_lib_target_name)',
         'deps/base64/base64.gyp:base64',
-        'deps/googletest/googletest.gyp:gtest',
-        'deps/googletest/googletest.gyp:gtest_main',
+        #'deps/googletest/googletest.gyp:gtest',
+        #'deps/googletest/googletest.gyp:gtest_main',
         'deps/histogram/histogram.gyp:histogram',
         'deps/uvwasi/uvwasi.gyp:uvwasi',
         'deps/simdutf/simdutf.gyp:simdutf',
@@ -1193,6 +1193,10 @@
         }],
         ['OS=="solaris"', {
           'ldflags': [ '-I<(SHARED_INTERMEDIATE_DIR)' ]
+        }],
+        # Skip cctest while building for OpenBSD
+        [ 'OS=="openbsd"', {
+          'type': 'none',
         }],
         # Skip cctest while building shared lib node for Windows
         [ 'OS=="win" and node_shared=="true"', {
